torch/_dynamo/variables/builtin.py

```diff
@@ -1428,6 +1429,20 @@ def call_set(self, tx: "InstructionTranslator", *args, **kwargs):
         else:
             unimplemented(f"set(): {args} {kwargs}")
 
+    def call_frozenset(self, tx: "InstructionTranslator", *args, **kwargs):
+        assert not kwargs
+        if not args:
+            return FrozensetVariable([])
+        assert len(args) == 1
+        arg = args[0]
+        if isinstance(arg, variables.FrozensetVariable):
+            return arg.clone()
```

Comment:
This isn't right.  This will clone the VariableTricker, which will keep things like `.source` intact and result in wrong aliasing relationships or return a non-frozenset.

Reply:
good catch.

Reply:
Thanks for catching this. I missed it.